R/standard-data-prep/update-geography-lookups.R

```diff
@@ -0,0 +1,98 @@
+library(readr)
+# @param open_geography_file Lookup file sourced from
```

Comment:
Is this intended as roxygen or just our own version? I think roxygen2 would need a ' after the # to work

Reply:
Was just using the roxygen's @param terminology to make a note really. Figured it wasn't worth doing full roxygen, but worth stealing some of the syntax! Happy to add the ' if you think we might start using roxygen in this repo.

Reply:
I'm pretty sure we will one day, and then turn this app into a package as well as the app so we can embed it in analysts pipeline but tbh it's fine to leave for now, we can add at later date.